librz/flag/flag.c

```diff
@@ -80,7 +80,7 @@ static void free_item_name(RzFlagItem *item) {
    dir == -1 -> result <= off
```

Comment:
If you make this function `RZ_API`, please convert the comment to the Doxygen format as well.

Reply:
I also suggest to rename `dir` to `direction` to be clearer about the purpose of it.